### ISSUE TITLE
[Bug]:  Sensors all gone to unknown

### ISSUE THREAD

**Issue by dunkelheim** [mampfes/hacs_waste_collection_schedule#1438]
### I Have A Problem With:

The integration in general

### What's Your Problem

After the integration works fine for me, it stops about three days ago. In the app store there was an update of the "bawn" app yesterday, till that time the App also didn't get the dates. Today the app works fine again but the HA sensors stay unknown on every type of garbage and the calender is empty too.  I downloaded the integration again / deleted + installed completly new without fixing the issue.

### Source (if relevant)

app_abfallplus_de

### Logs

```Shell
no logs from waste collection
```

### Relevant Configuration

```YAML
waste_collection_schedule:
    sources:
    - name: app_abfallplus_de
      args:
        app_id: de.k4systems.bawnapp
        city: xxx
        strasse: xxx
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [x] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by pille89** [comment]
I have the same problem, all sensors are unknown. 

Service provider tbv_velbert_de

**Configuration:**

```
waste_collection_schedule:
  sources:
    - name: tbv_velbert_de
      args:
        strasse: "Am Lindenkamp 33"
      customize:
        - type: Restmüll-Gefäß
          alias: rest
          icon: mdi:trash-can
        - type: Gelbe Tonne
          alias: gelbersack
          icon: mdi:recycle
        - type: Bio-Tonne
          alias: bio
          icon: mdi:trash-can
  fetch_time: "04:00"
  day_switch_time: "10:00"
  
sensor:
  - platform: waste_collection_schedule
    name: AbfallRestmuell
    details_format: "upcoming"
    value_template: '{{ value.daysTo }}'
    types:
      - rest

  - platform: waste_collection_schedule
    name: AbfallGelberSack
    details_format: "upcoming"
    value_template: '{{ value.daysTo }}'
    types:
      - gelbersack

  - platform: waste_collection_schedule
    name: AbfallBio
    details_format: "upcoming"
    value_template: '{{ value.daysTo }}'
    types:
      - bio
```

**Comment by 5ila5** [comment]
# @dunkelheim 
for me 

```yaml
waste_collection_schedule:
    sources:
    - name: app_abfallplus_de
      args:
        app_id: de.k4systems.bawnapp
        city: Drakenburg
        strasse: Grüne Straße
        hnr: 6
```
returns valid entries.



_I Guess for reainstalling you allrady did that . but for compleation_:

_Dates are only fetched once a day or when restarting. So if the dates where added today you may only see them tomorrow or after a restart._

Can you try this configuration (don't forget to restart HA) 

# @pille89:

> logs: source not found: tbv_velbert_de

tbv_velbert_de is not yet part of the current release but will be available in release 1.44.0 or now in the master (git) version (HACS: 3-dot menu -> re-download -> change version to master)

**Comment by pille89** [comment]
Hi, 

The master version has the same problem, I have already tried it.

**Comment by 5ila5** [comment]
# @pille89 

The error source not found: tbv_velbert_de means that the file is not there but the master version does contain this file. 
Did you fully restart HA  after changing the version?

note: as mentioned  in #1441 you need to use street not strasse argument

**Comment by dunkelheim** [comment]
@5ila5 I changed my configurations to yours and finaly got a log file. Same failure as I got at my first setup. Only had to change hnr: 6 in hnr: "6" and everything starts to work again. I then changed your settings to mine and it also worked again. I don't really know what changed to my settings (accedently deleted hnr in my first post), but as always, the biggest failure of a system seems to sit 20 cm away from the monitor. Thank you very much for helping. Greetings from Hassel.
@pille89 - is your problem fixed as well?
